src/stores/appStore.ts

```diff
@@ -27,7 +27,8 @@ export const useAppStore = defineStore("app", {
         admin1Indicators: {},
         admin1Geojson: {},
         admin2Indicators: {},
-        admin2Geojson: {}
+        admin2Geojson: {},
+        admin0Geojson: null
```

Comment:
Is there some reason why `admin1Geojson` and `admin2Geojson` are empty objects, while `admin0Geojson` is null?

Is it just so that you can write `if (admin0Geojson.value)` (because null is falsy and `{}` is truthy)?

I feel like the downside of implying that the property is of a different structure than its brethren is greater than the downside of writing `if (Object.keys(admin0Geojson.value) === 0)` or importing `isEmpty` from lodash: `if (isEmpty(admin0Geojson.value))`

Reply:
^ Wrote the above before I worked out that `admin0Geojson` had a different structure from the other `adminNGeojsons`: https://github.com/mrc-ide/arbomap/pull/17#discussion_r1589294467

Reply:
i could have gone for an object (or actually array here because the admin level 0 geojsons are a little different) however i feel like nulls are better than empty objects/arrays as initialisers because you can do falsy checks on them, you cant do those for empty objects and empty arrays

but this is all just preference!